install/kubernetes/cilium/templates/cilium-nodeinit-daemonset.yaml

```diff
@@ -61,8 +61,39 @@ spec:
           imagePullPolicy: {{ .Values.nodeinit.image.pullPolicy }}
           securityContext:
             privileged: true
-{{- if .Values.nodeinit.revertReconfigureKubelet }}
           lifecycle:
+{{- if and (.Values.eni.enabled) (not .Values.enableIPv4Masquerade) }}
+            postStart:
```

Comment:
This will run concurrently with the initialization of the container's entrypoint, which means that `iptables-save | grep -v AWS-SNAT-CHAIN | iptables-restore` will run at the same time the `STARTUP_SCRIPT` is executed. Is that ok?

Reply:
I _believe_ so, since in this case (ENI) what `STARTUP_SCRIPT` will do is...

1. ... mount the BPF filesystem (and installing the mount); and
1. ... outputting `ip link` and related command; ...

... so I'd say it's okay but I'm not 100% sure.